src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -769,6 +773,29 @@ class ArnoldAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 					// to get the behaviour people expect.
 					AiNodeSetArray( node, "trace_sets", AiArray( 1, 1, AI_TYPE_STRING, "__none__" ) );
 				}
+
+				if( m_linkedLights )
+				{
+					const std::vector<std::string> &lightNamesWritable = m_linkedLights->readable();
+
+					AtArray *linkedLightNodes = AiArrayAllocate( lightNamesWritable.size(), 1, AI_TYPE_NODE );
+					int i = 0;
+					for ( IECore::StringVectorData::ValueType::const_iterator it = m_linkedLights->readable().begin(); it != m_linkedLights->readable().end(); ++it, ++i )
+					{
+						// TODO: Can we avoid prefixing the path here?
```

Comment:
I don't know how you feel about that kind of stuff, but I had thought we could create some kind of utility method that does the prefixing, which we would then call from both https://github.com/GafferHQ/gaffer/blob/master/src/GafferArnold/IECoreArnoldPreview/Renderer.cpp#L1362 and from here. I understand the need for prefixing - I just thought it's a little ugly to duplicate the (admittedly small) logic for prefixing? If we ever need to change the prefix, we'd have to do it twice. Maybe I'm overly concerned about something that will never come up :P